packages/loaders/src/elements/Dots.tsx

```diff
@@ -61,18 +61,18 @@ export const Dots = forwardRef<SVGSVGElement, IDotsProps>(
 
     return (
       <StyledSVG
+        data-garden-id={COMPONENT_ID}
         ref={ref}
-        fontSize={size!}
-        color={color!}
+        $fontSize={size!}
+        $color={color!}
         width="80"
         height="72"
```

Comment:
I still think something about this has a bad smell. As a rule, we don't put `width`/`height` attributes on SVGs. Rather, the preference is to apply sizing via CSS. Not sure how/why these got here.